### ISSUE TITLE
sitewide 508-defect-3 [SCREENREADER]: Progress indicator SHOULD be consistent on the same page

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9226]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

```diff
! Team affected: VSA Caregiver; Project: 10-10CG MVP
```

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The progress indicator in the forms system has a bug that is evident for screen reader users. The individual steps indicate 50% progress, but the progress indicator associated with the headings communicates 33%. **[Here is a recording](https://dsva.slack.com/files/UMLC20J79/F013ELTQ8T1/progressindicatorissue-voiceover.m4a)** of the VoiceOver experience on Caregiver 10-10CG, on the step 2 of 4 Primary Family Caregiver information, Health care coverage.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to read the content of the page and receive consistent progress indicator communication.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging and production

## Steps to Recreate

1. Enter ~~https://staging.va.gov/caregivers/~~ https://staging.va.gov/family-member-benefits/comprehensive-assistance-for-family-caregivers/ in browser
2. Start screenreading device listed in Environment
3. Navigate through the form
4. Verify the progress indicator is inconsistent 
5. You may need to shift+tab to move focus to the h1 or the progress indicator, as the focus is currently on the h2 but should be on the h1

## Possible Fixes (optional)

Announce the progress in one place. It seems it is announced in two positions.

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Understanding, Consistent Identification SC 3.2.4](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

**Comment by timwright12** [comment]
Is there a new URL for this one? The caregivers one in the description seems to be 404ing

**Comment by jenstrickland** [comment]
@ala-yna Is this the correct new url for this in staging? https://staging.va.gov/family-member-benefits/comprehensive-assistance-for-family-caregivers/

**Comment by ala-yna** [comment]
Yes, that's correct @jenstrickland !

**Comment by taylorkaren** [comment]
Issue moved to [department-of-veterans-affairs/va-forms-system-core #251](https://github.com/department-of-veterans-affairs/va-forms-system-core/issues/251) via [**ZenHub**](https://www.zenhub.com/)
